client/src/components/ImageGenerator/functions.ts

```diff
@@ -1,5 +1,5 @@
-import { processPayment } from '../../utils/lightningUtils';
-import { generateImage, varyImage, upscaleImage } from '../../api/midjourneyApi';
+import { processPayment } from "../../utils/lightningUtils.js";
```

Comment:
You should delete the extra lightingUtils.ts file and change this line.

```suggestion
import { processPayment } from "../../utils/lightningUtils";
```